loadgen/loadgen.cc

```diff
@@ -904,17 +956,22 @@ bool PerformanceSummary::PerfConstraintsMet(std::string* recommendation) {
     case TestScenario::SingleStream:
       break;
     case TestScenario::MultiStream:
+      ProcessLatencies();
+      if (target_latency_percentile.query_intervals >= 2) {
```

Comment:
This is where the loadgen tests for the perf constraint of MultiStream:
After this patch, if more than (100-percentile)% of the frames produced result in 1 or more skipped intervals in the loadgen, the latency constraint will not be considered met.